cfn-templates/data-exports-aggregation.yaml

```diff
@@ -102,19 +110,24 @@ Conditions:
       - !Equals [!Ref 'AWS::Region', 'cn-northwest-1']
   ManageCUR2: !Equals [!Ref ManageCUR2, 'yes']
   ManageFOCUS: !Equals [!Ref ManageFOCUS, 'yes']
+  ManageCOH: !Equals [!Ref ManageCOH, 'yes']
   EnableSCAD: !Equals [!Ref EnableSCAD, 'yes']
   DeployDataExport:
     Fn::Or:
       - !Condition ManageCUR2
       - !Condition ManageFOCUS
+      - !Condition ManageCOH
   DeployCUR2ViaCFN:      !And [!Condition IsSourceAccount, !Condition ManageCUR2,        !Condition RegionSupportsDataExportsViaCFN]
   DeployFOCUSViaCFN:     !And [!Condition IsSourceAccount, !Condition ManageFOCUS,       !Condition RegionSupportsDataExportsViaCFN]
+  DeployCOHViaCFN:       !And [!Condition IsSourceAccount, !Condition ManageCOH,         !Condition RegionSupportsDataExportsViaCFN]
   DeployCUR2ViaLambda:   !And [!Condition IsSourceAccount, !Condition ManageCUR2,  !Not [!Condition RegionSupportsDataExportsViaCFN]]
   DeployFOCUSViaLambda:  !And [!Condition IsSourceAccount, !Condition ManageFOCUS, !Not [!Condition RegionSupportsDataExportsViaCFN]]
+  DeployCOHViaLambda:   !And [!Condition IsSourceAccount, !Condition ManageCOH,    !Not [!Condition RegionSupportsDataExportsViaCFN]]
   DeployCUR2Table:       !And [!Condition IsDestinationAccount, !Condition ManageCUR2]
   DeployFOCUSTable:      !And [!Condition IsDestinationAccount, !Condition ManageFOCUS]
-  DeployAnyExportViaLambda: !Or [!Condition DeployCUR2ViaLambda, !Condition DeployFOCUSViaLambda]
-  DeployAnyTable: !Or [!Condition DeployFOCUSTable, !Condition DeployCUR2Table]
+  DeployCOHTable:        !And [!Condition IsDestinationAccount, !Condition ManageCOH]
+  DeployAnyExportViaLambda: !Or [!Condition DeployCUR2ViaLambda, !Condition DeployFOCUSViaLambda, !Condition DeployCOHViaLambda]
```

Comment:
What is this new concept of any lambda?

Reply:
If any Export needs to be done via lambda - then need to deploy role and lambda